internal/app/list/list.go

```diff
@@ -52,14 +56,17 @@ func listConcurrently(item record.RecordEntity, dataFolder string) (chan string,
 	}
 
 	go func() {
-		result, errPrettyPrintDataFile := data.SPrettyPrintFile(
-			dataFolder+string(os.PathSeparator)+item.DataFileName,
+		defer close(c)
+
+		result, err := data.SPrettyPrintFile(
+			filepath.Join(dataFolder, item.DataFileName),
 			func(todo string, index int) string {
 				return fmt.Sprintf("\t%v. %v", index, todo)
 			},
 		)
-		if errPrettyPrintDataFile != nil {
-			c <- fmt.Sprintf("Error: Something went wrong: data.SPrettyPrintDataFile: %v", errPrettyPrintDataFile)
+		if err != nil {
+			c <- fmt.Sprintf("Error: Something went wrong: data.SPrettyPrintDataFile: %v", err)
+			return
```

Comment:
I didn't think you wanted to fall through here, sending probably-invalid data on the channel after sending this error message.